Service/ResponseTransformerStrategy/UnexpectedResponseStrategy.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Auto1\ServiceAPIClientBundle\Service\ResponseTransformerStrategy;
+
+use Auto1\ServiceAPIClientBundle\DTO\ResponseTransformer\ErrorResponse;
+use Auto1\ServiceAPIClientBundle\Exception\Response\DeserializableResponseException;
+use Auto1\ServiceAPIClientBundle\Service\Deserializer;
+use Auto1\ServiceAPIClientBundle\Service\ResponseTransformerStrategy;
+use Auto1\ServiceAPIComponentsBundle\Service\Endpoint\EndpointInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+use Psr\Log\NullLogger;
+use Symfony\Component\HttpFoundation\Response;
+
+class UnexpectedResponseStrategy implements ResponseTransformerStrategy, LoggerAwareInterface
```

Comment:
since this must be the last one of strategies, a priority should be set to ensure that it remains the last one even if in the future new strategies are added.

Reply:
added DefaultPriority to that class